hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/PrisonerSynchroniserService.kt

```diff
@@ -18,6 +18,7 @@ class PrisonerSynchroniserService(
   private val restrictedPatientService: RestrictedPatientService,
   private val incentivesService: IncentivesService,
   private val prisonerDifferenceService: PrisonerDifferenceService,
+  private val newOffenderEndpointTestService: NewOffenderEndpointTestService?,
```

Comment:
This is nullable because it's feature switched in each env - no feature switch => no test service => the tests are not run in that env.